File: Web/EvaMittal_EvaMittal_2125cs1187_2/GettingStarted/CountDigitsInANumber.java
import java.util.*;
public class CountDigitsInANumber {
    public static void main(String[] args) {
        // write your code here  
        Scanner sc = new Scanner(System.in);
        long n= sc.nextLong();
        sc.close();
        int d=0;
        while(n>0)
        {
            d++;
            n=n/10;
        }
        System.out.println(d);
       }
}

File: AR-VR/saumyasinha31_SaumyaSinha_2125cse1144_2/GettingStarted/Count Digits In A Number.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv){
    int n;
    cin >> n;
    int count,r;
    while(n!=0){
        r=n%10;
       
        count++;
         n=n/10;
        
    }
    cout<<count<<endl;
    //write your code here
    
}